xbmc/guilib/GUIComponent.cpp

```diff
@@ -41,12 +42,19 @@ CGUIComponent::~CGUIComponent()
 void CGUIComponent::Init()
 {
   m_pWindowManager->Initialize();
+  m_stereoscopicsManager.reset(new CStereoscopicsManager(CServiceBroker::GetSettings(),
+                                                         CServiceBroker::GetDataCacheCore(),
+                                                         *m_pWindowManager));
+  m_stereoscopicsManager->Initialize();
+
   CServiceBroker::RegisterGUI(this);
 }
 
 void CGUIComponent::Deinit()
 {
   CServiceBroker::UnregisterGUI();
+
+  m_stereoscopicsManager.reset();
```

Comment:
drop this. as mentioned, those are pointers just for one reason: forward refs in header